vnext/Microsoft.ReactNative/ReactHost/ReactInstanceWin.cpp

```diff
@@ -385,6 +390,43 @@ void ReactInstanceWin::LoadModules(
   }
 #endif
 
+  ::Microsoft::ReactNative::ExceptionsManager::SetRedBoxHander(
+      winrt::Microsoft::ReactNative::ReactPropertyBag(m_reactContext->Properties()), m_redboxHandler);
+  registerTurboModule(
+      L"ExceptionsManager",
+      winrt::Microsoft::ReactNative::MakeTurboModuleProvider<::Microsoft::ReactNative::ExceptionsManager>());
+
+  registerTurboModule(
+      L"StatusBarManager",
+      winrt::Microsoft::ReactNative::MakeModuleProvider<::Microsoft::ReactNative::StatusBarManager>());
+
+  registerTurboModule(
+      L"PlatformConstants",
```

Comment:
This will never get used.  

PlatformConstants gets registered as a cxxmodule here:
https://github.com/microsoft/react-native-windows/blob/ad17b41551aac833a917ae6e6ec7e7d78770cfaf/vnext/Shared/OInstance.cpp#L681

cxxmodules take precedence over turbomodules.

But you still need it as a cxxmodule when running Office on the non-abi code path

Reply:
Added a new `DevSettings` field to control the behavior.